easybuild/easyconfigs/m/MATLAB/MATLAB-2019a.eb

```diff
@@ -0,0 +1,25 @@
+name = 'MATLAB'
+version = '2019a'
+
+homepage = 'http://www.mathworks.com/products/matlab'
+description = """MATLAB is a high-level language and interactive environment
+ that enables you to perform computationally intensive tasks faster than with
+ traditional programming languages such as C, C++, and Fortran."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# be sure to copy both DVD content to the SAME directory, 
+# including the hidden files, especially .dvd1 and .dvd2 
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('Java', '1.8')]
```

Comment:
did you check that the Java dep is actually used?
see #7574
if not, I would add this line (or remove the Java dep):
modextrapaths = {'MATLAB_JAVA': '$EBROOTJAVA/jre'}

Reply:
@smoors You're right. I forgot about that. We need to decide which route to take to solve this.

Reply:
We should just take a decision on this. I tested it out with my 2019a install and saw no issues with either approach.

Reply:
I guess using the Java that ships with MATLAB (and hence dropping the `Java` dependency) is our safest bet, since even if `Java` gets loaded through some other module, it won't break anything (since the `MATLAB` installations have actually been using the internal Java until now).

Reply:
I agree with @boegel
@vanzod Want to drop the java dep?